Dockerfile

```diff
@@ -11,13 +11,8 @@ ADD clean-layer.sh  /tmp/clean-layer.sh
 ADD patches/nbconvert-extensions.tpl /opt/kaggle/nbconvert-extensions.tpl
 ADD patches/template_conf.json /opt/kaggle/conf.json
 
-# This is necessary for apt to access HTTPS sources
-RUN apt-get update && \
-    apt-get install apt-transport-https && \
```

Comment:
This package is already part of the `gcr.io/deeplearning-platform-release/base-cpu` image.